packages/client/src/runtime/RequestHandler.ts

```diff
@@ -28,10 +37,12 @@ export type RequestParams = {
   modelName?: string
   protocolMessage: ProtocolMessage
   dataPath: string[]
+  debugPanic?: 'fatal' | 'non-fatal'
```

Comment:
This is only implemented in binary engine and not in the two others (which are the main ones), can we maybe remove?